keyboards/0118/blacktip/rules.mk

```
@@ -0,0 +1,10 @@
+MCU_FAMILY 				 	= STM32
+MCU_SERIES 				 	= STM32L0xx
+MCU_LDSCRIPT 			 	= STM32L072xZ
+MCU_STARTUP 			 	= stm32l0xx
+MCU						 	= cortex-m0plus
+ARMV 					 	= 6
+BOARD 					 	= ST_NUCLEO64_L073RZ
+
+BOOTLOADER				 	= stm32-dfu
+STM32_BOOTLOADER_ADDRESS 	= 0x1FF00000
```

Shouldn't have a mix of tabs and spaces. 
```suggestion
MCU_FAMILY = STM32
MCU_SERIES = STM32L0xx
MCU_LDSCRIPT = STM32L072xZ
MCU_STARTUP = stm32l0xx
MCU	= cortex-m0plus
ARMV = 6
BOARD = ST_NUCLEO64_L073RZ

STM32_BOOTLOADER_ADDRESS 	= 0x1FF00000
```

Bootloader can/should be moved to json.